lib/phlex/renderable.rb

```diff
@@ -8,7 +8,10 @@ def render(renderable, *args, **kwargs, &block)
 					block = Phlex::Block.new(self, &block)
 				end
 
+				@_original_wrapper = @_wrapper
+				@_wrapper = renderable
 				renderable.call(@_target, view_context: @_view_context, parent: self, &block)
+				@_wrapper = @_original_wrapper
```

Comment:
I’m not convinced it should be `@wrapper` relative if we do this. https://github.com/joeldrapper/phlex/pull/252#issuecomment-1277362082